daybreak_install.sh

```diff
@@ -0,0 +1,86 @@
+#! /bin/bash
+
+set -e
+
+COMPILER_BACKENDS="gcc clang zig"
+
+function _choose_compiler_backend() {
+	if [ $# -eq 0 ]; then
+		echo "Could not find a supported compiler backend!"
+		echo "Please install and make available on PATH one of the following:"
+		echo ${COMPILER_BACKENDS}
+		exit 1
+	fi
+
+	CC=$1
+	shift
+	echo "Checking for compiler backend ${CC}"
+	if ! command -v ${CC} &> /dev/null; then
+		echo "Missing compiler backend ${CC}"
+		set_compiler $@
+	fi
+}
+
+function bootstrap_install() {
+	DAYBREAK_BOOTSTRAP=${DAYBREAK_SOURCE_FOLDER}/bootstrap/${OS}/daybreak
+	DAYBREAK_OUTPUT=${DAYBREAK_HOME}/bin/daybreak
+	if [[ ${OS} = "windows" ]]; then
+		DAYBREAK_BOOTSTRAP=${DAYBREAK_BOOTSTRAP}.exe
+		DAYBREAK_OUTPUT=${DAYBREAK_OUTPUT}.exe
+	fi
+	C_INCLUDE_PATH=${DAYBREAK_SOURCE_FOLDER}/src/include
+	${DAYBREAK_BOOTSTRAP} ${DAYBREAK_SOURCE_FOLDER}/src/main.day -o ${DAYBREAK_OUTPUT}
+}
+
+function print_export_env() {
+	echo "Add the following to your rcfile of choice to preserve the installed configuration:"
```

Comment:
I still think automating this path environment setup out would be good but if we really don't want to mess with root privileges on our end at all then we should improve this message to let the user know they NEED to make these changes if they want to run daybreak without fully qualifying the path name.

Reply:
I tried actually running this with a `source` or `.` so it would inject the env vars, but I really don’t like scripts which automatically add to your rcfiles.

instead providing instruction is better. Homebrew gives an almost identical message on install.

An improvement that could be made is showing the command to add to your rcfile with an echo.

Reply:
+1 for the echo command. Would be nice to just copy and paste one thing and type in my password.

Reply:
Voila!
![Screenshot_2022-07-21_21-36-03](https://user-images.githubusercontent.com/17893076/180363483-05dfc9cc-2725-4f3a-a0ad-445c96d0d54b.png)